packages/augment-api/src/interfaces/extraDefs/definitions.ts

```diff
@@ -1,5 +1,9 @@
 /* eslint-disable */
-// add any additional definitions (rpc etc) here
+
+import { latest, runtime } from '@kiltprotocol/type-definitions'
+
+// Only types and runtime calls can be exported from here.
```

Comment:
what's the reason for having to split it in two files?

Reply:
So the RPC endpoints must be decorated in a folder that has the same name as the module you are trying to augment. So for `api.rpc.did` stuff, they have to be declared in a `did` folder. The rest, meaning types and runtime APIs, can be declared in the same folder, with no naming requirements. It's explained in [this section](https://polkadot.js.org/docs/api/examples/promise/typegen#types-setup)